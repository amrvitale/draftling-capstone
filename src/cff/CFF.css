* {
    box-sizing: border-box;
  }

  body {
      margin: 0;
  }

  .critique {
    margin: 1px;
    padding: 5px;
    border: 1px solid maroon;
    border-radius: 5px;
    background-color: black;
    justify-content: space-evenly;
    font-size: 20px;
  }



  p {
    white-space: pre-wrap; 
}