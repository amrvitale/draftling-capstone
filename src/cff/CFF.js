import React from 'react'
import CritiqueFreeform from '../critiquefreeform/CritiqueFreeform';
import CritiqueTemplate from '../critiquetemplate/CritiqueTemplate'

class CFF extends React.Component {
    render() {
        let critiqueType;

       // if ( s ) {
       // critiqueType = <CritiqueFreeform />
      //  }

       // else  {
       //  critiqueType = <CritiqueTemplate />
       // }
       
        return (
            <div className="critiqueList">
              {this.props.critiqueType}
            </div>
        );
       
    }
};

export default CFF;

//if else like with status button, using column as flag