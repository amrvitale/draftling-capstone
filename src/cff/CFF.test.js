import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import CFF from './CFF';

describe (`CFF component`, () => {
    it('renders a .CFF by default', () => {
        const wrapper = shallow(<CFF crit={{opening: "lots of comments", critfreeform: "quite interesting"}}/>)
        expect (toJson(wrapper)).toMatchSnapshot()
    })

    
})