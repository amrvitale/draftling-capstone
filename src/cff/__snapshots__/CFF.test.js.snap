// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CFF component renders a .CFF by default 1`] = `
<div
  className="critique"
>
  <h3>
    Freeform Critique
  </h3>
  <p
    className="critText"
  >
    Opening comments: 
    lots of comments
  </p>
  <br />
  <p
    className="critText"
  >
    Overall: 
    quite interesting
  </p>
</div>
`;
